README.md

```diff
@@ -28,5 +28,205 @@ This is a proposed template repository for applications for future developments.
 1. Click the `Code` button to clone the repository and follow instructions to clone to your local machine.
 1. Navigate to the directory on your local machine.
 1. Install dependencies with `npm install`.
-1. Add `env.development` and `.env.production` file to the root directory of your project. Both should follow the exact structure of the `/.env.template` file. Fill in your secrets.
+1. Add `env.development` file to the root directory of your project. This file should follow the structure of the `/.env.template` file. Fill in your secrets.
 1. Run the Gatsby development server `gatsby develop` or `npm run dev`.
+
+## Project Structure
+
+### Files & Folders You Should Edit
```

Comment:
It seems like this is almost the same as the documentation page. So if you needed to make changes here, you would also have to make changes there. 

Instead, should we get rid of everything from this line forward (with the exception of Troubleshooting) to avoid repetition and include only the most important information here? We can have a note at the end of the README directing to the documentation page for more info.